test/flows/student_finance_calculator_flow_test.rb

```diff
@@ -0,0 +1,519 @@
+                    whats_your_household_income?: "50,000",
+                    do_any_of_the_following_apply_uk_full_time_students_only?: "children-under-17",
+                    what_course_are_you_studying?: "teacher-training"
+    end
+
+    should "render text for when the student is eligible for extra grants and allowances" do
+      assert_rendered_outcome text: "Depending on your income and circumstances"
+    end
+
+    should "render text for when the student is not eligible for extra grants and allowances" do
+      add_responses do_any_of_the_following_apply_uk_full_time_students_only?: "no",
+                    what_course_are_you_studying?: "none-of-the-above"
+
+      assert_rendered_outcome text: "You don’t qualify for extra grants and allowances"
+    end
+
+    should "render text for when the student is eligible for childcare grant, one child" do
+      add_responses whats_your_household_income?: "15,000"
+
+      assert_rendered_outcome text: "a week for a single child"
+    end
+
+    should "render text for when the student is eligible for childcare grant, more than one child" do
+      add_responses whats_your_household_income?: "25,000"
+
+      assert_rendered_outcome text: "a week in Childcare Grant if you have 2 or more children"
+    end
+
+    should "render text for when the student is eligible for parents learning allowance" do
+      add_responses whats_your_household_income?: "18,000"
+
+      assert_rendered_outcome text: "Parents’ Learning Allowance"
+    end
+
+    should "render text for child tax credit when the student has a child under 17" do
+      assert_rendered_outcome text: "Child Tax Credit"
+    end
+
+    should "render text for when the student is eligible for an adult dependant's grant" do
+      add_responses whats_your_household_income?: "15,000",
+                    do_any_of_the_following_apply_uk_full_time_students_only?: "dependant-adult"
+
+      assert_rendered_outcome text: "Adult Dependant’s Grant"
+    end
+
+    should "render text if student has a disability" do
+      add_responses do_any_of_the_following_apply_uk_full_time_students_only?: "has-disability"
+
+      assert_rendered_outcome text: "Disabled Students’ Allowances"
+    end
+
+    should "render text if student has low income" do
+      add_responses do_any_of_the_following_apply_uk_full_time_students_only?: "low-income"
+
+      assert_rendered_outcome text: "University and college hardship funds"
+    end
+
+    should "render text if student is eligible for teacher training funding" do
+      assert_rendered_outcome text: "Funding for teacher training"
+    end
+
+    should "render text if student is eligible for social work funding" do
+      add_responses what_course_are_you_studying?: "social-work"
+
+      assert_rendered_outcome text: "Social Work Bursary"
+    end
+
+    should "render the full time student text within the extra help partial" do
```

Comment:
As long as we're testing the `if` statement in that partial in one of the other tests we can delete this test. If we're not testing the `if` somewhere else we can change this to look for the full time text